README.md

```diff
@@ -1 +1,22 @@
-# ussp-validation
\ No newline at end of file
+# USSP Test Suite Implementation
+
+This repository contains BR-UTM's implementation of a validation tool for UAS Service Provider (USP) implementation compliance with standards and regulations and it's functionalities.
+
+## Standards and Regulations
+
+Compliance with the following standards and regulations:
+
+- [ASTM F3411-19](https://www.astm.org/f3411-19.html);
```

Comment:
We don't expect to support ASTM F3411-19, only the F3411-22 one

Reply:
Done.